.frogbot/frogbot-config.yml

```diff
@@ -0,0 +1,12 @@
+- params:
+    git:
+      repoName: frogbot
+      branches:
+        - dev
+    scan:
+      projects:
+        - workingDirs:
+            - .
+        - installCommand: npm i
+          workingDirs:
+            - action
```

Comment:
There is a problem with the 'action' directory and the postinstall script. As you can see here, when scanning this dir it offers a fix with too many changes in the node_modules:

https://github.com/jfrog/frogbot/pull/215

It's better not to scan it right now until we figure out how to handle it.